apps/wing-console/console/ui/src/ui/custom-resource-ui-field.tsx

```diff
@@ -1,21 +1,34 @@
+import { ShareIcon } from "@heroicons/react/24/outline";
 import { Attribute } from "@wingconsole/design-system";
+import { Link } from "@wingconsole/design-system";
 
 import { trpc } from "../services/trpc.js";
 
 export interface CustomResourceUiFieldItemProps {
   label: string;
   handlerPath: string;
+  link: boolean;
 }
 
 export const CustomResourceUiFieldItem = ({
   label,
   handlerPath,
+  link,
 }: CustomResourceUiFieldItemProps) => {
   const field = trpc["uiField.get"].useQuery(
     {
       resourcePath: handlerPath,
     },
     { enabled: !!handlerPath },
   );
-  return <Attribute name={label} value={field.data?.value ?? ""} />;
+  const fieldValue = field.data?.value ?? "";
+  return (
+    <Attribute name={label} value={link ? undefined : fieldValue}>
+      {link && (
+        <Link id={label} href={fieldValue}>
+          <ShareIcon title={fieldValue} className="w-4 h-4 text-blue-500" />
+        </Link>
```

Comment:
```suggestion
        <Link id={label} href={fieldValue} target="_blank">
          {fieldValue}
        </Link>
```

Reply:
![Screenshot 2024-05-06 at 13 40 42](https://github.com/winglang/wing/assets/2538825/624d6572-969b-43c6-a373-e75dab04aa9d)

I think this is better